### PR TITLE
Update templates for dotnet isolated

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

File: Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs [hunk 1]
```diff
@@ -11,7 +9,7 @@ public static class TimerTriggerCSharp
         [FunctionName("TimerTriggerCSharp")]
```
Comment (mhoeger) [inline Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs:9]: Function?

**Review by fabiocav — APPROVED** [review]

File: Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs (hunk 1 above)
Comment (fabiocav) [reply Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs:9]: Oops! Missed this ! Good catch.

**Review by fabiocav — CHANGES_REQUESTED** [review]

File: Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs (hunk 1 above)
Comment (ankitkumarr) [reply Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs:9]: Thank you thank you! Looks like I missed here and in Service Bus.

File: Functions.Templates/Templates/ServiceBusQueueTrigger-CSharp-Isolated/ServiceBusQueueTriggerCSharp.cs [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
-using Microsoft.Azure.Functions.Worker.Extensions.ServiceBus;
 using Microsoft.Extensions.Logging;
 
 namespace Company.Function
 {
     public static class ServiceBusQueueTriggerCSharp
     {
-        [FunctionName("ServiceBusQueueTriggerCSharp")]
+        [FunctionN("ServiceBusQueueTriggerCSharp")]
```
Comment (ankitkumarr) [inline Functions.Templates/Templates/ServiceBusQueueTrigger-CSharp-Isolated/ServiceBusQueueTriggerCSharp.cs:9]: omg

**Review by fabiocav — APPROVED** [review]

**Pull request merged**